OSDOCS-16919: CQA fixes for developer perspective/catalog customizati… (#117200)
openshift/openshift-docs: modules/odc_con_example-yaml-file-changes.adoc

[id="con_example-yaml-file-changes_{context}"]
= Example YAML file changes

[role="_abstract"]

> You can dynamically add the following snippets in the YAML editor for customizing a developer catalog.

Reviewer: Short desc _[Must not include self referential language](https://docs.google.com/spreadsheets/d/1qXFRcYZncrAex1fuRjOzlAHB2X3_WHAeVxffY1GfL50/edit?gid=503904153#gid=503904153)_. Not sure how to rewrite this one. WDYT?

```suggestion
You can customizing a developer catalog by dynamically edit the console YAML content in the YAML editor.
```

Reviewer: Also, I know this isn't a CQA thing, but should this module go right after [Customizing a developer catalog or its sub-catalogs using the YAML view](https://117200--ocpdocs-pr.netlify.app/openshift-enterprise/latest/web_console/customizing-the-web-console.html#odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-yaml-view_customizing-web-console)? These examples can be applied using that procedure, right?
